compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/TypeChecker.java

```diff
@@ -1023,6 +1023,12 @@ public void visit(BLangInvocation iExpr) {
                         iExpr.expr.type);
                 resultType = symTable.semanticError;
                 break;
+            case TypeTags.TYPEDESC:
+                if (dlog.errorCount == 0) {
```

Comment:
This may not be a scalable solution. What if there were other errors logged previously that are not related to the type desc?